interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/emails/telehealth-invitation-existing.html.twig

```diff
@@ -0,0 +1,171 @@
+ # @link      http://www.open-emr.org
+ # @author    Stephen Nielson <snielson@discoverandchange.com>
+ # @copyright Copyright (c) 2023 Comlink Inc <https://comlinkinc.com/>
+ # @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+<!DOCTYPE html
+        PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml"
+      xmlns:o="urn:schemas-microsoft-com:office:office">
+
+<head>
+    <title>{{ title|text }}</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0 " />
+    <meta name="format-detection" content="telephone=no" />
+    <!--[if !mso]><!-->
+    <link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,700,700i,900,900i"
+          rel="stylesheet" />
+    <!--<![endif]-->
+    <style type="text/css">
+        body {
+            margin: 0;
+            padding: 0;
+            -webkit-text-size-adjust: 100% !important;
+            -ms-text-size-adjust: 100% !important;
+            -webkit-font-smoothing: antialiased !important;
+        }
+
+        img {
+            border: 0 !important;
+            outline: none !important;
+        }
+
+        p {
+            margin: 0px 0px 20px 0px !important;
+            padding: 0px !important;
+        }
+
+        table {
+            border-collapse: collapse;
+            mso-table-lspace: 0px;
+            mso-table-rspace: 0px;
+        }
+
+        td,
+        a,
+        span {
+            border-collapse: collapse;
+            mso-line-height-rule: exactly;
+        }
+    </style>
+</head>
+
+<body class="em_body" style="margin:0px auto; padding:0px;" bgcolor="#efefef">
+<table width="100%" border="0" cellspacing="0" cellpadding="0" class="em_full_wrap" align="center"
+       bgcolor="#efefef">
+    <tr>
+        <td align="center" valign="top">
+            <table align="center" width="650" border="0" cellspacing="0" cellpadding="0" class="em_main_table"
+                   style="width:650px; table-layout:fixed;">
+                <tr>
+                    <td align="center" valign="top" style="padding:0 25px;" class="em_aside10">
+                        <table width="100%" border="0" cellspacing="0" cellpadding="0" align="center">
+                            <tr>
+                                <td height="26" style="height:26px;" class="em_h20">&nbsp;</td>
+                            </tr>
+                            <tr>
+                                <td align="center" valign="top"><a href="#" target="_blank"
+                                                                   style="text-decoration:none;"><img
+                                                src="{{ logoPath|attr_url }}"
```

Comment:
wouldn't put entire url paths through url (really just meant for get parameters within the url string), so would just use `attr` above

Reply:
Fixed